agent/agent-bootstrap/src/main/java/com/microsoft/applicationinsights/agent/bootstrap/diagnostics/status/StatusFile.java

```diff
@@ -62,33 +65,16 @@ public class StatusFile {
   // visible for testing
   static final String FILE_EXTENSION = ".json";
 
-  // visible for testing
-  static final String SITE_LOGDIR_PROPERTY = "site.logdir";
```

Comment:
just confirming, this isn't used any more? I thought I remember seeing -Dsite.logdir=... before in app services, but maybe not

Reply:
I don't see it though. where did you see it before? was it for linux or windows? linux defaults everything to /var/log/applicationinsights.

Reply:
can you search the windows app service AI extension source to see if it mentions site.logdir?